syntax = "proto3";

message test_msg {
  string test_string = 1;
  int32 test_int = 2;
}
